CHANGELOG.md

```diff
@@ -52,13 +52,17 @@ v1.9.0 [unreleased]
 -	[#21358](https://github.com/influxdata/influxdb/pull/21358): fix: disable MergeFiltersRule until it is more stable
 -	[#21516](https://github.com/influxdata/influxdb/pull/21516): fix: FGA enablement
 
-v1.8.6 [unreleased]
+v1.8.6 [2021-05-21]
 -------------------
 
+Note: No OSS specific changes were made in this release.
+
 -	[#21290](https://github.com/influxdata/influxdb/pull/21290): fix: Anti-Entropy loops endlessly with empty shard
```

Comment:
if there are no OSS specific fixes...why are these three PRs listed?

Reply:
The commits needed to be made to the OSS repo as a result of existing code organization.